github.js

```diff
@@ -79,17 +79,32 @@ GitHub.listWorkflowsForRepo = async function listWorkflowsForRepo(repoName, repo
     }
 };
 
+GitHub.getUsage = async function getUsage(repoOwner, repoName, workflowId, run_id) {
+    try {
+        let usage = await octokit.actions.getWorkflowRunUsage({
+            repo: repoName,
+            owner: repoOwner,
+            workflow_id: workflowId,
+            run_id: run_id
+        });
+        return usage;
+    } catch (e) {
+        console.error('Error getting usage', e);
+        return {};
+    }
+};
 
 GitHub.getMostRecentRuns = async function getMostRecentRuns(repoOwner, repoName, workflowId) {
     try {
         const sevenDaysAgo = dayjs().subtract(7, 'day');
         const runs = await octokit.paginate(octokit.actions.listWorkflowRuns, { repo: repoName, owner: repoOwner, workflow_id: workflowId });
         if (runs.length > 0) {
             const groupedRuns = _.groupBy(runs, 'head_branch')
-            const rows = _.reduce(groupedRuns, (result, runs, branch) => {
+            const rows = _.reduce(groupedRuns, async (result, runs, branch) => {
```

Comment:
Isn't this missing an await on the _.reduce?

Also won't this run serially?  Feels like it would be better to do a bounded parallel get of the usage instead of looking at it serially